doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1431,7 +1431,7 @@ The command line option ``-R`` is a variant of ``-Q`` which has the strictly
 same behavior regarding loadpaths, but which also makes the
 corresponding ``.vo`` files available through their short names in a way
 not unlike the ``Import`` command (see :ref:`here <import_qualid>`). For instance, ``-R`` `path` ``Lib``
```

Comment:
In passing, we might want to simplify the formatting on the line above.

```suggestion
not unlike the ``Import`` command (see :ref:`here <import_qualid>`). For instance, ``-R path Lib``
```

Reply:
@gshen42 If you meant to apply the suggestion, it did not work (or you didn't push maybe) and you only managed to mark this comment as "resolved".